wpimath/util/__init__.py

```diff
@@ -0,0 +1,18 @@
+from .units import (
```

Comment:
These shouldn't be imported here. If anything, you'd want `from . import units`. With these here, a user can do:

```
wpimath.util.degreesToRadians(deg)
```

Or 

```
wpimath.util.units.degreesToRadians(deg)
```

Which... they're both a bit weird honestly. I feel like I'd prefer them being in `wpimath/units.py` so then you can do:

```
wpimath.units.degreesToRadians(deg)
```
I feel like @auscompgeek would have an opinion on this.